server/models/Collective.js

```diff
@@ -905,13 +905,7 @@ export default function(Sequelize, DataTypes) {
    * It does **not** check that the collective is indeed a host.
    */
   Collective.prototype.canApply = async function() {
```

Comment:
You can also remove `async`.

Reply:
We used to have an eslint rule for that, not sure what happened to it.